examples/Example19/SplitKernels.cuh

```diff
@@ -0,0 +1,259 @@
+// SPDX-FileCopyrightText: 2022 CERN
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef SPLITKERNELS_CUH
+#define SPLITKERNELS_CUH
+
+#include "SplitKernels.h"
```

Comment:
Please rename to `example19.{cuh,h}` (or `example.{cuh,h}` if you want something more generic)

Reply:
Generic is better for (future) diffs, thanks!